packages/account/src/Sections/Security/SelfExclusion/self-exclusion.jsx

```diff
@@ -550,8 +531,9 @@ class SelfExclusion extends React.Component {
                                                                 value={values.max_turnover}
                                                                 onChange={handleChange}
                                                                 onBlur={handleBlur}
+                                                                maxLength={this.getMaxLength('max_turnover')}
                                                                 required
-                                                                error={touched.max_turnover && errors.max_turnover}
+                                                                error={errors.max_turnover}
```

Comment:
Why remove the form `touched` event? Do we want to validate immediately without a focus / touched event?

Reply:
Actually I think `touched` event does not work as intended. It does not validate fields on focus, it will only validate fields when you touch and focus outside of the touched field. Because of this currently on production, users do not see validation messages before the field loses focus which was already raised for another field in the system and previously for binary as well. As for removal, without `touched` it works exactly the way as it should and it doesn't validate fields before users focus on it.

Reply:
Ok, let's go ahead with this then.